settings.py

```diff
@@ -76,7 +76,11 @@ def get_setting(setting):
     elif setting in strings:
         return str(utils.ADDON.getSetting(setting))
     elif setting in paths:
-        return translatePath(utils.ADDON.getSetting(setting).encode("utf-8"))
+        try: 
+            path = utils.ADDON.getSetting(setting).encode("utf-8")
+        except UnicodeDecodeError:
```

Comment:
in fact I had errors with "é" caracter in exclusion path.
Adding this exception make the job in this case